ple/games/donkeykong/board.py

```diff
@@ -31,16 +31,16 @@ def __init__(self, width, height, rewards, rng, dir):
         self._dir = dir
 
         self.IMAGES = {
-        	"background": pygame.image.load(os.path.join(dir, 'Assets/background.png')).convert_alpha(),
-        	"still": pygame.image.load(os.path.join(dir, 'Assets/still.png')).convert_alpha(),
-        	"kong0": pygame.image.load(os.path.join(dir, 'Assets/kong0.png')).convert_alpha(),
-        	"princess": pygame.image.load(os.path.join(dir, 'Assets/princess.png')).convert_alpha(),
-        	"heart": pygame.image.load(os.path.join(dir, 'Assets/heart.png')).convert_alpha(),
-        	"board": pygame.image.load(os.path.join(dir, 'Assets/board.png')).convert_alpha(),
-        	"fireballright": pygame.image.load(os.path.join(dir, 'Assets/fireballright.png')).convert_alpha(),
-        	"coin1": pygame.image.load(os.path.join(dir, 'Assets/coin1.png')).convert_alpha(),
-        	"wood_block": pygame.image.load(os.path.join(dir, 'Assets/wood_block.png')).convert_alpha(),
-        	"ladder": pygame.image.load(os.path.join(dir, 'Assets/ladder.png')).convert_alpha()
+        	"background": pygame.image.load(os.path.join(dir, 'assets/background.png')).convert_alpha(),
```

Comment:
if the file does not use alpha, like the background call convert() instead of convert_alpha(). convert_alpha() should only be used it there is transparency in the image